client/src/components/Login.style.js

```diff
@@ -0,0 +1,85 @@
+import styled from 'styled-components';
+
+export const StyleAll = styled.div`
```

Comment:
I think we should try and use less ```div```s and more semantic HTML. This means that this StyleAll could instead be something like a ```<main>```